js/main.js

```diff
@@ -55,6 +63,19 @@ $homeBtn.addEventListener('click', function () {
   switchViews('home');
 });
 
+var $favesBtn = document.querySelector('.faves-nav');
+var noFaves = document.querySelector('.no-faves');
+$favesBtn.addEventListener('click', function () {
+  switchViews('faves');
+  if (data.favoriteGames.length === 0) {
+    noFaves.className = 'no-faves view';
+  } else {
+    noFaves.className = 'no-faves hidden';
+    renderFavedGames();
+  }
```

Comment:
Optionally, I'd suggest looking at the `classList` API for a less manual way of adding / removing individual classes.

Reply:
Okay, thank you! I'll take a look into it.